cli/src/main/java/io/aklivity/zillabase/cli/internal/commands/start/ZillabaseStartCommand.java

```diff
@@ -1155,17 +1155,20 @@ private static final class CreateStudioFactory extends CreateContainerFactory
         CreateStudioFactory(
             ZillabaseConfig config)
         {
-            super(config, "studio", "ghcr.io/aklivity/zillabase/zillabase-studio:%s".formatted(config.studio.tag));
+            super(config, "studio", "ghcr.io/aklivity/zillabase/studio:%s".formatted(config.studio.tag()));
         }
 
         @Override
         CreateContainerCmd createContainer(
-            DockerClient client)
+            DockerClient client,
+            ZillabaseConfig config)
```

Comment:
We don't need to change the signature to pass `config` to `createContainer(...)` method, as `config` is already visible as a field from the `CreateContainerFactory` super class, populated by constructor.